docs/system-admin-guide/users-permissions/roles-permissions/README.md

```diff
@@ -7,91 +7,135 @@ keywords: manage roles, manage permissions
 
-You will see the list of all the roles that have been created so far.
+The Anonymous role can not be deleted. 
 
-**Non member** is the default role of users of your OpenProject instance who have not been added to a project. This only applies if the project has been set as **public** in the project settings.
-**Anonymous** is the default role of users who are not logged in. This only applies if you disabled the need for authentication for your instance and if the project is set as **public**.
-Those two roles can't be deleted.
+## Customize roles and their permissions 
 
-![create roles](system-guide-roles.png)
+Administrators can customize the roles and the permissions of the different roles in **Administration** -> **Users and permissions** -> **Roles and permissions**.
 
-After clicking the green **+ Role** button, a form will be shown to define the role and its permissions.
+### Permissions report
+
+The permissions report is a good starting point to get and overview of the current configuration. 
+
+### Create a new project roles
+
+Administrators can create new project roles in **Administration** -> **Users and permissions -> Roles and permissions**.
 
 Complete the following as required:
 
 1. **Role name** - must be entered and be a new name.
-2. **Global Role** - this role applies to all projects, and can be assigned in the [user details](../users/#manage-user-settings) or in the [group details](../groups/#add-global-roles-to-a-group). Once saved, the decision to make a role global can't be reverted.
-   Ticking this box will show the available [global roles](#global-roles) and hide the regular permission options.
 3. **Copy workflow from** - select an existing role. The respective [workflows](../../manage-work-packages/work-package-workflows) will be copied to the role to be created.
 4. **Permissions** for this role - you can specify the permissions per OpenProject module. Click the arrow next to the module name to expand or compress the permissions list.
 
-Select the permissions which should apply for this role. You can use **check all** or **uncheck all** at the right of a module permissions list. If a module is not enabled in a project it is not shown to a user despite having a permission for it.
+Select the permissions which should apply for this role.
+
+> **Note:** If a module is not enabled in a project it is not shown to a user despite having a permission for it.
+
+### Create a new global role
+
+Administrators can create new global role in **Administration** -> **Users and permissions -> Roles and permissions**.
+
+To create a global roles activate the Global role check box in the create form.
+
+### Edit and delete roles
+
+To delete an existing role click on the **delete icon** next to a role in the list. 
+
+> **Note:**  Roles that are assigned to a user can not be deleted. 
+
+## FAQ
+
+### When should I use the permission "Edit users"?
```

Comment:
Hint: With [this ticket](https://community.openproject.org/projects/openproject/work_packages/48967) we split the Edit users permission into "edit users" and "create users". If the administrator wants to fully delegate user management, both global permissions need to be given.